Management/Mapping/DomainToStorageMapper.cs

```diff
@@ -13,5 +13,14 @@ public static Common.StorageModels.User ToStorage(User domain)
                 CreatedAt = domain.CreatedAt.ToString(),
                 PassportId = domain.PassportId.Value,
             };
+        public static Management.StorageModels.Comment ToStorage(Comment domain) => new Management.StorageModels.Comment
+        {
```

Comment:
I feel null check is not really necessary in here? When `DomainModels.Comment` is created, we already have the null check in ctor.

Reply:
You will never know which part can break...

Add nullable check itself does not hurt and it will prevent the unknown excpetion thrown to a large extent.

Reply:
Okay, I can add the null checks.